esmvaltool/cmorizers/data/formatters/datasets/noaa_ersstv5.py

```diff
@@ -0,0 +1,126 @@
+"""ESMValTool CMORizer for NOAA ERSST data, version 5.
+
+   This is the CMORizer script for the NOAA Extended Reconstructed Sea Surface
+   Temperature (ERSST) data of version 5.
+
+Tier
+    Tier 2: open dataset.
+
+Source
+    https://doi.org/10.7289/V5T72FNM
+
+Last access
+    20200520
+
+Download and processing instructions
+    The data is available via an ftp-server provided by NOAA. The script
+    below can be used to automate the process.
```

Comment:
Why is this script needed if we already have the download functions implemented?

Reply:
Yes true! I had left the comments from what was in there before. I have removed that bash script now. Thanks